src/apps/chifra/pkg/rpcClient/get_log.go

```diff
@@ -92,7 +92,7 @@ func GetLogsByTransactionId(chain string, bn, txid uint64) ([]types.SimpleLog, e
 		Txid:    txid,
 		NeedsTs: true,
 		Ts:      blockTs,
-	})
+	}, nil)
```

Comment:
Same as above. An alternative would be to use some sort of global from the package, `NoStore` or something.